.github/workflows/actions.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Checkout the code
         uses: actions/checkout@v4
       - name: Run the formatter, linter, and vetter
-        uses: dell/common-github-actions/go-code-formatter-linter-vetter@main
+        uses: dell/common-github-actions/go-code-formatter-linter-vetter@testgomodule
```

Comment:
Why was the change made here? GitHub Actions are stull failing.

Reply:
It will be switched to main, once we fix the errors in GitHub actions, it seems like the error is coming from GitHub action rather than the changes introduced in this PR.